chips/msp432/src/lib.rs

```diff
@@ -0,0 +1,140 @@
+#![crate_name = "msp432"]
+#![crate_type = "rlib"]
+#![feature(asm, const_fn, in_band_lifetimes)]
```

Comment:
`in_band_lifetimes` have been removed from Tock, can you avoid re-introducing them?

Reply:
Sure, I'll remove it!